README.brazilian-portuguese.md

```diff
@@ -1166,6 +1166,6 @@ Nossos colaboradores são membros que estão contribuindo com o repositório em
 
 ## Thank You Notes
 
-We appreciate any contribution, from a single word fix to a new best practice. View our contributors and [contributing documentation here!](CONTRIBUTORS.md)
+We appreciate any contribution, from a single word fix to a new best practice. View our contributors and [contributing documentation here!](./.operations/CONTRIBUTING.md)
```

Comment:
These changes (and similar in the other README files) happens because the CONTRIBUTORS page was merged to the main README.md file so instead I linked the contribution guide, if wanted I can change the link to the readme file and the Contributors section